balancer/weightedroundrobin/weightedroundrobin.go

```diff
@@ -16,16 +16,20 @@
  *
  */
 
-// Package weightedroundrobin defines a weighted roundrobin balancer.
+// Package weightedroundrobin provides an implementation of the weighted round
+// robin LB policy, as defined in gRFC A58:
+// https://github.com/grpc/proposal/blob/master/A58-client-side-weighted-round-robin-lb-policy.md
```

Comment:
Easwar gave me go style comment about how urls should be formatted. See https://github.com/grpc/grpc-go/pull/6165#discussion_r1166083658.